src/plugins/event-plugins/DomEventPlugin.ts

```diff
@@ -31,10 +31,12 @@ export type PartialDomEventPluginConfig = {
 };
 
 export type DomEventPluginConfig = {
+    allowDynamicDomEventListeners?: boolean;
```

Comment:
issue: Let's consider some alternatives for the config option name:
- `enableDynamicUpdates`
- `enableMutationObserver`

Reply:
Will refactor to use `enableMutationObserver`